### PR TITLE
ACS-11290 Cascading Dictionaries Home bootstraping

### PR DESCRIPTION
**Pull request by gerardolenski**
More information in ticket: https://hyland.atlassian.net/browse/ACS-11290

### CODE REVIEW

File: repository/src/main/resources/alfresco/patch/patch-services-context.xml [hunk 1]
```diff
@@ -174,6 +174,27 @@
         </property>
     </bean>
 
+    <bean id="patch.patch.cdDefinitionsSpace" class="org.alfresco.repo.admin.patch.impl.GenericBootstrapPatch" parent="basePatch">
```
[the remark this answers is not in the document]
Comment (gerardolenski) [reply repository/src/main/resources/alfresco/patch/patch-services-context.xml]: typo, thanks

File: repository/src/test/java/org/alfresco/repo/admin/patch/CDDefinitionPatchIT.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    @Value("${spaces.company_home.childname}")
+    private String companyHomeChildName;
+    @Value("${spaces.dictionary.childname}")
+    private String dictionaryChildName;
+    @Value("${spaces.cd_definitions.childname}")
+    private String cdDefinitionsChildName;
+
+    @Test
+    public void testCDPatchConfiguration()
```
Comment (b-ansari) [inline repository/src/test/java/org/alfresco/repo/admin/patch/CDDefinitionPatchIT.java:66]: Minor suggestion that, can we Add a test that invokes the patch directly a second time and asserts no duplicate folder is created?

e.g.  testCDPatchIsIdempotent()
    assertThat(refs).hasSize(1); // Still exactly one

**Review by b-ansari — APPROVED** [review]

File: repository/src/test/java/org/alfresco/repo/admin/patch/CDDefinitionPatchIT.java (hunk 2 above)
Comment (gerardolenski) [reply repository/src/test/java/org/alfresco/repo/admin/patch/CDDefinitionPatchIT.java:66]: Yep, it is good idea

**Review by b-ansari — APPROVED** [review]
